pkg/analyser/analyzer_test.go

```diff
@@ -940,23 +990,31 @@ func TestAnalyze(t *testing.T) {
 				}
 			}
 
-			unmanagedChanges, err := diff.Diff(result.Unmanaged(), c.expected.Unmanaged())
-			if err != nil {
-				t.Fatalf("Unable to compare %+v", err)
-			}
-			if len(unmanagedChanges) > 0 {
-				for _, change := range unmanagedChanges {
-					t.Errorf("%+v", change)
+			for i, expected := range c.expected.Unmanaged() {
```

Comment:
diff have an ooption to check slice ordering: 
>SliceOrdering ensures that the ordering of items in a slice is taken into account
https://github.com/r3labs/diff

Reply:
Yes indeed! thanks